tox.ini

```diff
@@ -11,4 +11,6 @@ python =
 
 [testenv]
 extras = testing
-commands = python -m pytest --mpl -v --color=yes --cov=napari_matplotlib --cov-report=xml
+commands =
+    - python -c 'from skimage import data; data.brain()'
```

Comment:
Given this import and call is in `conftest.py`, I'm not sure why it needs to be here too. Do the tests fail without this line in `tox.ini`?